distributed/utils.py

```diff
@@ -13,6 +14,7 @@
 import re
 import shutil
 import socket
+import time
```

Comment:
from .metrics import time

The windows `time.time` function has a 1s precision limit.